source/manual/govuk-env-sync.html.md

```diff
@@ -1,11 +1,11 @@
 ---
 owner_slack: "#re-govuk"
-title: "Datasync with govuk_env_sync"
+title: Environment data sync
 section: Monitoring
 type: learn
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2020-09-15
+last_reviewed_on: 2020-11-04
```

Comment:
No longer need this (or the line below) - sorry! 😁

Reply:
Ah yes, I've removed them now!